impl/sql/mutationstorage/queue.go

```diff
@@ -175,24 +175,48 @@ func (m *Mutations) HighWatermark(ctx context.Context, directoryID string, logID
 func (m *Mutations) ReadLog(ctx context.Context, directoryID string,
```

Comment:
Might make sense to point out that this method can return more than `batchSize` entries for timestampy reasons.